### PR TITLE
Fix export of CSV.

### PR DESCRIPTION
**Pull request by stratease**
[TEC-4442](https://theeventscalendar.atlassian.net/browse/TEC-4442)

Fix issues with CSV exports for the MEC data. The settings page ID had changed, and was not updated here in it's reference in the export URL.

### CODE REVIEW

File: tribe-ext-export-mec-events.php [hunk 1]
```diff
@@ -24,6 +24,8 @@
  */
 
 // Do not load unless Tribe Common is fully loaded and our class does not yet exist.
+use Tribe\Events\Admin\Settings;
+
```
Comment (Camwyn) [inline tribe-ext-export-mec-events.php:28]: Does $this->add_required_plugin( 'Tribe__Events__Main' ); (below, line 41, in unedited code) require a TEC version set?

File: tribe-ext-export-mec-events.php [hunk 2]
```diff
@@ -381,34 +411,33 @@ public function venues_csv_setup() {
 		/**
 		 * Generate the CSV files.
 		 *
+		 * @since TBD Fixed warning from order of header output.
+		 *
 		 * @param string $csv_file_name - The name of the CSV file to be created
-		 * @param array  $header        - The name of the columns
-		 * @param array  $data
+		 * @param array  $csv_header    - The name of the columns
```
Comment (Camwyn) [inline tribe-ext-export-mec-events.php]:
```suggestion
		 * @param array  $csv_header    The name of the columns
```

File: readme.txt [hunk 3]
```diff
@@ -35,6 +35,12 @@ We're always interested in your feedback and our [premium forums](https://theeve
 
 == Changelog ==
 
+= [1.0.1] TBD =
+
+* Compatibility - Setting required The Events Calendar version to 5.15.0. [TEC-4442]
```
Comment (Camwyn) [inline readme.txt]:
```suggestion
* Compatibility - Setting the required version of The Events Calendar to 5.15.0. [TEC-4442]
```

File: tribe-ext-export-mec-events.php [hunk 4]
```diff
@@ -381,34 +411,33 @@ public function venues_csv_setup() {
...
 			header( "Content-Disposition: attachment; filename={$csv_file_name}" );
 			header( 'Expires: 0' );
 			header( 'Pragma: public' );
 
-			fputcsv( $fh, $header );
+			// Write the file header for correct encoding ( UTF8 ).
+			fprintf( $fh, chr( 0xEF ) . chr( 0xBB ) . chr( 0xBF ) );
 
+			// Output CSV data.
+			fputcsv( $fh, $csv_header );
```
Comment (Camwyn) [inline tribe-ext-export-mec-events.php:434]:
```suggestion
			fputcsv( $fh, $csv_header );
			
```
I added a line break

File: tribe-ext-export-mec-events.php [hunk 5]
```diff
@@ -125,7 +153,9 @@ public function listen_for_export_button() {
 
 			if ( empty( $_REQUEST['export_events'] ) && empty( $_REQUEST['export_organizers'] ) && empty( $_REQUEST['export_venues'] ) || ! wp_verify_nonce( $_REQUEST['_wpnonce'], 'export_events' ) && ! wp_verify_nonce( $_REQUEST['_wpnonce'], 'export_organizers' ) && ! wp_verify_nonce( $_REQUEST['_wpnonce'], 'export_venues' ) ) {
```
Comment (Camwyn) [inline tribe-ext-export-mec-events.php:154]: Not a changed line, but this feels terribly redundant?

Comment (Camwyn) [reply tribe-ext-export-mec-events.php:154]: ...and long 😉

File: readme.txt (hunk 3 above)
Comment (Camwyn) [reply readme.txt]: slightly better wording 😉

**Review by Camwyn — CHANGES_REQUESTED** [review]
Couple small wording/format changes. The others are observations for a possible future refactor/reorganization (you can ignore those for the purposes of the review 😉).

File: tribe-ext-export-mec-events.php (hunk 5 above)
Comment (stratease) [reply tribe-ext-export-mec-events.php:154]: Reads like Shakespeare!

**Review by Camwyn — APPROVED** [review]

**Pull request merged**